cplusplus.json

```diff
@@ -813,5 +813,145 @@
+  "examples": ["https://github.com/casacore/casacore/tree/master/msfits/apps"]},
+  {"id": "cpp.qt-verbal-expressions", "name": "QtVerbalExpressions", "tech": ["utils"], "tags": ["regex"], "repo": "VerbalExpressions/QtVerbalExpressions",
+  "imports": ["QVerbalExpressions.h"], "examples": ["https://github.com/VerbalExpressions/QtVerbalExpressions/tree/master/example"]},
+  {"id": "cpp.libcsptr", "name": "C Smart Pointers", "tech": ["faster-code"], "tags": ["smart-pointer"], "repo": "Snaipe/libcsptr",
+  "imports": ["csptr/"], "examples": ["https://github.com/Snaipe/libcsptr/tree/master/check/test"]},
+  {"id": "cpp.cxx-prettyprint", "name": "cxx-prettyprint", "tech": ["eng-tools"], "tags": ["pretty-output"], "repo": "louisdx/cxx-prettyprint",
+  "imports": ["prettyprint.hpp"], "examples": ["https://github.com/louisdx/cxx-prettyprint/blob/master/ppdemo.cpp"]},
+  {"id": "cpp.gcc-poison", "name": "gcc-poison", "tech": ["reliable-code"], "tags": ["unsafe-functions"], "repo": "leafsr/gcc-poison",
+  "imports": ["poison.h"], "examples": ["https://github.com/leafsr/gcc-poison#example-usage"]},
+  {"id": "cpp.kangaru", "name": "kangaru", "tech": ["app-arch"], "tags": ["inversion-of-control"], "repo": "gracicot/kangaru",
+  "imports": ["kangaru/kangaru.hpp"], "examples": ["https://github.com/gracicot/kangaru/tree/master/examples"]},
+  {"id": "cpp.klib", "name": "Klib", "tech": ["data-structures"], "tags": ["collections"], "repo": "attractivechaos/klib",
+  "imports": ["bgzf.h", "kbit.h", "keigen.h", "khash.h", "kmath.h", "", "kseq.h", "kstring.h", "kurl.h", "kalloc.h", "kbtree.h", "kexpr.h",
+  "khmm.h", "knetfile.h", "kson.h", "ksw.h", "kvec.h", "kavl.h", "kdq.h", "kgraph.h", "klist.h", "knhx.h", "ksort.h", "kthread.h"],
+  "examples": ["https://github.com/attractivechaos/klib/tree/master/test"]},
+  {"id": "cpp.libsigcpp", "name": "libsigc++", "tech": ["app-arch"], "tags": ["callback"], "repo": "GNOME/libsigcplusplus",
+  "imports": ["sigc++/sigc++.h"], "examples": ["https://github.com/GNOME/libsigcplusplus/tree/master/examples"]},
+  {"id": "cpp.libcpuid", "name": "libcpuid", "tech": ["utils"], "tags": ["cpu"], "repo": "anrieff/libcpuid",
+  "imports": ["libcpuid.h"]},
+  {"id": "cpp.libevil", "name": "libevil", "tech": ["app-blocks"], "tags": ["license"], "repo": "avati/libevil",
+  "imports": ["libevil.h"]},
+  {"id": "cpp.libnih", "name": "libnih", "tech": ["data-structures"], "tags": ["collections"], "repo": "keybuk/libnih",
+  "imports": ["nih/"], "examples": ["https://github.com/keybuk/libnih/tree/master/nih/tests"]},
+  {"id": "cpp.libusb", "name": "libusb", "tech": ["peripheral"], "tags": ["usb"], "repo": "libusb/libusb",
+  "imports": ["libusb.h"], "examples": ["https://github.com/libusb/libusb/tree/master/examples"]},
+  {"id": "cpp.mach7", "name": "Mach7", "tech": ["diagnostics"], "tags": ["monitoring"], "repo": "solodon4/Mach7",
```

Comment:
something went wrong here. solodon4/Mach7 is pattern matching of some sort.

Reply:
yes it should be tech=="app-arch", tags==["design-patterns"]